lib/client/api.go

```diff
@@ -3458,6 +3465,33 @@ func (tc *TeleportClient) getSSHLoginFunc(pr *webclient.PingResponse) (SSHLoginF
 	}
 }
 
+// previewPlatformPasswordless provides indirection for tests.
+var previewPlatformPasswordless = wancli.PreviewIfPlatformPasswordlessRegistered
```

Comment:
Call this one `hasCredentials` too? (or whatever it is that we end up calling the others.)